lisa/tools/mount.py

```diff
@@ -73,7 +73,7 @@ class Mount(Tool):
 
     # /dev/da1p1 on /mnt/resource (ufs, local, soft-updates)
     _partition_info_regex_bsd = re.compile(
-        r"\s*/dev/(?P<name>.*)\s+on\s+(?P<mount_point>.*)\s+"
+        r"\s*(?:\/dev\/|zroot\/)(?P<name>.*)\s+on\s+(?P<mount_point>.*)\s+"
```

Comment:
Provide new examples above, and make sure there is no regression on existing patterns.

Reply:
Tested the changes on all the freebsd image versions there is no regression and I have added an example below the existing one

Reply:
I tested above examples on https://regex101.com/. It looks the new case doesn't return the mount point. 

```suggestion
        r"\s*(?:\/dev\/|zroot\/)(?P<name>.*)\s+on\s+(?P<mount_point>.*?)\s+"
```

Reply:
I also tested it on regex101.com and it works. The regex does check for a newline as the end of the capture for the mount point so if you just copy and paste the examples from the code into regex101.com without a trailing newline it will cause some weird behavior with the second example. Would it be helpful to add a newline character to the end of the examples so that it is more explicit?